apps/discovery-platform/src/entry-server/index.ts

```diff
@@ -73,27 +76,37 @@ export const entryServer = (ops: {
     return res.json({ quota: createdQuota });
   });
 
-  apiRouter.get("/request/entry-node", async (req, res) => {
+  apiRouter.post("/request/entry-node", async (req, res) => {
     const { client } = req.body;
+    log.verbose("requesting entry node for client", client);
+    if (typeof client !== "string") throw Error("Client is not a string");
+
     // check if client has enough quota
     const doesClientHaveQuotaResponse = await doesClientHaveQuota(
       ops.db,
       client,
       ops.baseQuota
     );
-    if (!doesClientHaveQuotaResponse) {
+    // TODO: reenable
+    if (false && !doesClientHaveQuotaResponse) {
```

Comment:
lets re-enable, we should fund a client in the E2E tests and update the quota

Reply:
should we fund them as part of the sandbox scripts or will funding be manual on every sandbox run?